### ISSUE TITLE
Unable to preventDefault inside passive event listener due to target being treated as passive.

### ISSUE THREAD

**Issue by HaysaRodrigues** [davidtheclark/react-aria-modal#79]
Hello!

I am trying to use the react-aria-modal, I'm copying my code below so you can see how it's implemented, with all the properties I'm using. But I am facing the issue **`Unable to preventDefault inside passive event listener due to target being treated as passive.`**, when I try to click outside the dialog.

```
 <AriaModal
        onExit={this.props.onClose}
        titleText="Add to Bag"
        underlayClickExits
        escapeExits
        underlayColor="none"
        includeDefaultStyles={false}
      >
```

does anyone knows how to fix this?

**Comment by HaysaRodrigues** [comment]
> Hello!
> 
> I am trying to use the react-aria-modal, I'm copying my code below so you can see how it's implemented, with all the properties I'm using. But I am facing the issue **`Unable to preventDefault inside passive event listener due to target being treated as passive.`**, when I try to click outside the dialog.
> 
> ```
>  <AriaModal
>         onExit={this.props.onClose}
>         titleText="Add to Bag"
>         underlayClickExits
>         escapeExits
>         underlayColor="none"
>         includeDefaultStyles={false}
>       >
> ```
> does anyone knows how to fix this?

this one has solved, https://github.com/OwlCarousel2/OwlCarousel2/issues/1884, but I don't know if it is the best solution.
